app/bundles/InstallBundle/Command/InstallCommand.php

```diff
@@ -292,31 +324,31 @@ protected function execute(InputInterface $input, OutputInterface $output)
             }
         }
 
-        if (isset($allParams['site_url']) && !empty($allParams['site_url'])) {
+        if (!empty($allParams['site_url'])) {
```

Comment:
`empty()` first checks if the variable is set, so the `isset()` is superfluous here. That's why I removed it. See https://www.php.net/manual/en/function.empty.php for details.